templates/games/chameleon_game.html

```diff
@@ -25,21 +27,23 @@
         </div>
         <div id="game">
             <div class="game-window">
-              <div id="chameleon-game" onclick="change_color('chameleon-game')">
-                This is the chameleon. (I know, it's a beauty, eh?)
-              </div>
+			<span id="scores" style="color:#b76e79;font-size:125%;">Your Current Score: 0</span>
+              <div id="chameleon-game"> 
+				<div class="fui-github" style="color:#7F8C8D;font-size:230%;position:relative;top:104%;" onclick="change_color('chameleon-game', 'fui-github')"></div>
```

Comment:
In real world code, its best to not inline css junk and instead move it to chameleon_game.css. Meh, not that important to fix.